client/my-sites/earn/memberships/index.tsx

```diff
@@ -608,8 +649,7 @@ function MembershipsSection( { query } ) {
 					</div>
 					<div>
 						<h3>
-							{ englishLocales.includes( getLocaleSlug() ) ||
-							i18n.hasTranslation( 'Simple fees structure' )
+							{ englishLocales.includes( locale ) || i18n.hasTranslation( 'Simple fees structure' )
```

Comment:
You can remove the `hasTranslation` check; I added it originally when this line was merged and translations had not finished — but I forgot to clean up after.

You can [check translations status](https://translate.wordpress.com/deliverables/overview/8026738/) but generally anything older than 1+ weeks has been translated.

Reply:
Great. Removed both conditional translations in this file.